### DISCUSSION TITLE
gatsby develop freezes on fresh setup

### DISCUSSION THREAD

**Discussion by damdafayton** [gatsbyjs/gatsby#36357] [Help] [answered]
Its the first time Im using gatsby and my fresh install freezes like this:
```
> gatsby develop

⠸ compile gatsby files
```

Ive never seen any framework doesnt work on the first run except Rails. It really killed my motivation to use it, since Im having very negative first experience.

There is no error or log. It just freezes.

I have: 
- Windows 10
- Node v16.15.1
- Gatsby CLI version: 4.20.0

### What I did?

I have tried with fresh installs of javascript and typescript versions of Gatsby.
I have installed with
`gatsby new app` and `gatsby new` commands.

I did 
```
gatsby clean
gatsby develop
```
and
```
gatsby clean
gatsby run develop or gatsby run build
```

All have frozen in similar way.

**Comment by TylerBarnes** [comment, marked as answer]
Hi @damdafayton sorry to hear your first experience with Gatsby didn't go so well!
I think this would be better as an issue rather than a discussion. Can you create an issue instead and provide reproduction steps (everything you did prior to running `gatsby develop`). I haven't heard of other users running into this so I wonder if it's something specific to your machine. It would be good to figure it out either way.

**Reply by damdafayton** [reply]
@TylerBarnes I am not sure how to reproduce it because there is nothing else you do before `gatsby develop` other than `npm install`. It just doesnt work. I decided not to use Gatsby anymore. Its incredible that it just doesnt start after the install. And documentation is very misleading. They suggest to use WSL2 if you dont want to waste time with windows. And I managed to run it on WSL2 but its increadibly slow due to file system differences. My thought is that Gatsby team doesnt care about windows users.

**Reply by TylerBarnes** [reply]
@damdafayton that's unfortunate you decided not to use it. Gatsby definitely cares about windows support, many of our engineers use windows.
